src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.TestResultService/BonusPointsRule.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DevAdventCalendarCompetition.TestResultService.Interfaces;
+
+namespace DevAdventCalendarCompetition.TestResultService
+{
+    public class BonusPointsRule : ITestResultPointsRule
+    {
+        private const int MaxBonus = 30;
```

Comment:
it should be readonly

Reply:
It is a constant integer not a field initialized in a constructor, so I think it should be const.

Reply:
my bad ;p